core/pkg/server/handler.go

```diff
@@ -167,43 +167,27 @@ func (h *Handler) Do(inChan <-chan *spb.Record) {
 		h.handleRecord(record)
 
 		switch record.RecordType.(type) {
-		case *spb.Record_Alert:
-			h.fwdRecord(record)
-		case *spb.Record_Artifact:
-			h.fwdRecord(record)
-		case *spb.Record_Config:
-			h.fwdRecord(record)
-		case *spb.Record_Files:
-			h.fwdRecord(record)
-		case *spb.Record_History:
-			h.fwdRecord(record)
-		case *spb.Record_Metric:
-			h.fwdRecord(record)
-		case *spb.Record_Output:
-			h.fwdRecord(record)
-		case *spb.Record_OutputRaw:
-			h.fwdRecord(record)
-		case *spb.Record_Preempting:
-			h.fwdRecord(record)
-		case *spb.Record_Stats:
-			h.fwdRecord(record)
-		case *spb.Record_Summary:
-			h.fwdRecord(record)
-		case *spb.Record_Telemetry:
-			h.fwdRecord(record)
-		case *spb.Record_UseArtifact:
+		default:
```

Comment:
lets move to the bottom?

Reply:
I would say it's better to have the exceptions at the bottom and the normal case at the top. It's different in spirit from using `default` as a fallback, which is why it's normally at the bottom.

Reply:
yes, but i think it is more golang idiomatic so i would still think it's better at the bottom could add a comment if you want to clarify that this is the default behavior

Reply:
i think the keyword `default` already implies it's the default behavior :) and anyway this is a common pattern:

* https://mtsknn.fi/blog/switch-statements-default-doesnt-have-to-be-the-last-case/#normal-case-first
* https://docs.datadoghq.com/code_analysis/static_analysis_rules/go-best-practices/switch-default-first-or-last/ (see the second "good" example)

Reply:
if that makes you happy